app/build.gradle

```diff
@@ -19,6 +19,7 @@ android {
         buildConfigField "String", "USERS_ENDPOINT", '"/users"'
         buildConfigField "String", "ITEMS_ENDPOINT", '"/items"'
         buildConfigField "String", "LOGIN_ENDPOINT", '"/login"'
+        buildConfigField "String", "DATE_PATTERN", "dd.MM.yy"
```

Comment:
The String `"dd.MM.yy"` needs to be wrapped inside single quotes '', so that kotlin recognizes this as a string

Reply:
Should be resolved in 3b2aba5.